llvm/lib/Transforms/IPO/MemProfContextDisambiguation.cpp

```diff
@@ -2073,14 +2073,14 @@ std::string IndexCallsiteContextGraph::getLabel(const FunctionSummary *Func,
                                                 unsigned CloneNo) const {
   auto VI = FSToVIMap.find(Func);
   assert(VI != FSToVIMap.end());
+  auto CallerName = getMemProfFuncName(VI->second.name(), CloneNo);
```

Comment:
nit: Spelling out the type as std::string makes it easier to reason about the operations that follow.

Reply:
done